packages/db/src/connection.ts

```diff
@@ -0,0 +1,129 @@
+import Knex from 'knex';
+import { getSecretConnectionConfig, getEnvConnectionConfig } from './config';
+
+import { envConectionConfigObject, knexSecretConnectionConfigObject } from './types';
+
+export interface knexConnectionConfigObject {
+  host: string,
+  username: string,
+  password: string,
+  database?: string,
+}
+
+interface knexConfigObject {
```

Comment:
I don't think this is necessary if we reference `Knex.Config` instead.

Reply:
Agreed.   5cc93d287